CentralHub.Api.Model/MeasurementCollection.cs

```diff
@@ -0,0 +1,25 @@
+
+using System.ComponentModel.DataAnnotations;
+using Microsoft.EntityFrameworkCore;
+
+namespace CentralHub.Api.Model;
+
+[Keyless]
+public sealed class MeasurementCollection
```

Comment:
i agree, initially we wanted to pass `int trackerId` and `List<Measurement>` to a method in RoomRepository, but it seems the http framework here don't support multiple parameters.

Otherwise maybe sending a jsonobject with one entry, the id as key and a list of measurements as value? I've tried to get that to work but i can't get it to work properly

Reply:
finished implementing it as a dictionary with list values of measurement instead.